golem-worker-service-base/src/gateway_binding/mod.rs

```diff
@@ -37,17 +37,25 @@ mod worker_binding_compiled;
 pub enum GatewayBinding {
     Default(WorkerBinding),
     FileServer(WorkerBinding),
-    Static(StaticBinding),
+    Static(Box<StaticBinding>),
```

Comment:
Why do we need the box here?

Reply:
Clippy.

Reply:
Due to size it holds. So basically clippy driven change.